client/bloodhound/transport.go

```diff
@@ -0,0 +1,97 @@
+package bloodhound
```

Comment:
This code is unchanged, other than exporting and adding the `roundtripperFunc` as it was moved from `utils.go`